website/content/en/docs/building-operators/helm/migration.md

```diff
@@ -1,12 +1,10 @@
 ---
 title: Migrating Helm-based Legacy Projects
 linkTitle: Migrating Legacy Projects
-weight: 300
-description: Instructions for migrating a legacy Helm-based project to use the new Kubebuilder-style layout.
+weight: 3
+description: This guide walks through an example of migrating a simple nginx-operator which was built by following the [legacy quick-start][quickstart-legacy] to the new Kubebuilder-style layout.
```

Comment:
This doc is pretty generic and doesn't really use the legacy sample project as an example.

Let's keep the description as is:
```suggestion
description: Instructions for migrating a legacy Helm-based project to use the new Kubebuilder-style layout.
```